androidApp/src/main/java/dev/johnoreilly/confetti/sessions/SessionListView.kt

```diff
@@ -129,6 +130,7 @@ fun SessionListView(
     }
 }
 
+private val dateFormatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM)
```

Comment:
I didn't add it inside Koin as we may want to use different formats for different screens - and it seems an overkill to use qualifiers and an`get<>()` inside the composable for that.

If you prefer to move there, let me know.

Reply:
Shouldn't we converge on dev.johnoreilly.confetti.utils.DateService? Or move away from that?

Maybe add some new simple methods there?

Reply:
I had a look into `DataService` and it seems we can cover all its use cases with a few extension functions inside common targeting `kotlinx.datetime` and providing an implementation per platform.

I will fill a bug for follow-up work.